tests/common/utilities.py

```diff
@@ -1137,7 +1137,7 @@ def capture_and_check_packet_on_dut(
         wait_time: the time to wait before stopping the packet capture, default is 1 second
     """
     pcap_save_path = "/tmp/func_capture_and_check_packet_on_dut_%s.pcap" % (str(uuid.uuid4()))
-    cmd_capture_pkts = "sudo nohup tcpdump --immediate-mode -U -i %s -w %s >/dev/null 2>&1 %s & echo $!" \
```

Comment:
Because we want to use this function on ptfhost, and ptfhose can't use sudo, so I removed sudo, by the way, sudo is not neccessary on duthost, so the remove won't affect other tests, I verified it on testbed.